packages/utilities/src/rtl.test.ts

```diff
@@ -1,18 +1,14 @@
-import { setSSR } from './dom/setSSR';
+export {};
```

Comment:
Remove this line?

Reply:
it can't be removed because of `--isolatedModules`